packages/neos-ui-editors/src/Editors/Image/index.js

```diff
@@ -210,10 +210,13 @@ export default class ImageEditor extends Component {
     }
 
     handleChooseFromMedia = () => {
-        const {secondaryEditorsRegistry} = this.props;
+        const {secondaryEditorsRegistry, options} = this.props;
         const {component: MediaSelectionScreen} = secondaryEditorsRegistry.get('Neos.Neos/Inspector/Secondary/Editors/MediaSelectionScreen');
 
-        this.props.renderSecondaryInspector('IMAGE_SELECT_MEDIA', () => <MediaSelectionScreen type="images" onComplete={this.handleMediaSelected}/>);
+        // set media type constraint to "image/*" if it is not explicitly specified via options.constraints.mediaTypes
+        const constraints = {...options.constraints, mediaTypes: (options.constraints && options.constraints.mediaTypes) || ['image/*']};
+
+        this.props.renderSecondaryInspector('IMAGE_SELECT_MEDIA', () => <MediaSelectionScreen constraints={constraints || {}} onComplete={this.handleMediaSelected}/>);
```

Comment:
Here we still should keep the `type` for now as mentioned in the other PR.

Reply:
Ah, keep it on the _calling_ side, too. Good point, and adjusted!